src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
Yes, it is in the PAL.a, but the problem is that it will be omitted from the libcoreclr.so